lab_1_classify_by_unigrams/main.py

```diff
@@ -149,11 +276,23 @@ def collect_profiles(paths_to_profiles: list) -> list[dict[str, str | dict[str,
 
     In case of corrupt input arguments, None is returned
     """
+    is_corrupt_input = isinstance(paths_to_profiles, list)
+    if not is_corrupt_input:
+        return None
+    collection_of_profiles: list[dict[str, str | dict]] | None
+    collection_of_profiles = []
+    for path in paths_to_profiles:
+        profile: dict[str, str | dict[str, float]] | None = load_profile(path)
+        if isinstance(profile, dict):
+            preprocessed_profile: dict[str, str | dict] | None = preprocess_profile(profile)
+            if preprocessed_profile is not None:
+                collection_of_profiles.append(preprocessed_profile)
+    return collection_of_profiles
 
 
 def detect_language_advanced(
-    unknown_profile: dict[str, str | dict[str, float]], known_profiles: list
-) -> list | None:
+        unknown_profile: dict[str, str | dict[str, float]], known_profiles: list
+) -> list[tuple[str, float]] | None:
```

Comment:
Is it wriong typing in the stub?

Reply:
fixed, there were some problems with this one because of my local MyPy